helpers/http_client.py

```diff
@@ -0,0 +1,41 @@
+from typing import Dict, List, Optional, Union
+
+import requests
+from requests import Response
+
+from helpers.http_session import http
+
+
+class HTTPClient:
+    """
+    Wrapper class on top of requests library
+    Serves as unwrapper on each HTTP request to avoid boilerplate request.status_code checking
+
+    All method signatures are matching respective requests.HTTP_method()
+
+    For example, self.get(self, url, **kwargs) is mathching requests.get(self, url, **kwargs)
+    signature
+
+    Implement more HTTP methods as needed
+    """
+    def __init__(self, http_session: requests.Session):
+        self.http = http_session
+        assert type(self.http) == requests.Session
+
+    @staticmethod
+    def _unwrap_response(response_obj: Response) -> Optional[Union[Dict, List]]:
+        if response_obj.ok:
+            return response_obj.json()
```

Comment:
What happens here if response_obj.ok is not True?  Should there be an explicit return None or is that implicit?

Reply:
If there is no explicit return in the function, None will be returned by default.